backends/tfhe-cuda-backend/cuda/src/integer/integer.cuh

```diff
@@ -456,9 +456,9 @@ void host_propagate_single_carry(cudaStream_t *streams, uint32_t *gpu_indexes,
   }
 
   cudaSetDevice(gpu_indexes[0]);
-  host_radix_blocks_rotate_right(streams, gpu_indexes, gpu_count, step_output,
-                                 generates_or_propagates, 1, num_blocks,
-                                 big_lwe_size);
+  host_radix_blocks_rotate_left(streams, gpu_indexes, gpu_count, step_output,
+                                generates_or_propagates, 1, num_blocks,
+                                big_lwe_size);
```

Comment:
why change the rotation direction here?

Reply:
Ok I got it wrong, I reverted all these changes and changed the PR name and description.